### PR TITLE
fix: update docs after merging build commands

### PR DESCRIPTION
**Pull request by mfocko**
Related to packit/packit#1611

Signed-off-by: Matej Focko <mfocko@redhat.com>

### CODE REVIEW

File: content/docs/CLI/build/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: build
+date: 2022-07-15
+chapter: false
+weight: 40
+bookCollapseSection: true
+---
```
Comment (TomasTomecek) [inline content/docs/CLI/build/_index.md:7]: Could you please put some content here, ideally links to the subsections as one can actually navigate here and sees a blank page, i.e. http://localhost:1313/docs/cli/build/

**Review by TomasTomecek — APPROVED** [review]
Otherwise LGTM, well done!

File: content/docs/CLI/build/_index.md (hunk 1 above)
Comment (mfocko) [reply content/docs/CLI/build/_index.md:7]: K

**Review by TomasTomecek — APPROVED** [review]
👏🏻

**Pull request merged**